portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,32 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import com.google.gson.*;
+import com.google.sps.data.Comments;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
+  private final Comments comments = new Comments();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello George</h1>");
+    response.setContentType("application/json");
+    String json = new Gson().toJson(comments);
```

Comment:
I think the Gson can be a private final variable, instead of being instantiated for every call.

Reply:
Done.